packages/css-library/src/stylesheets/base/_va.scss

```diff
@@ -0,0 +1,374 @@
+@import '../override-function';
+
+@import 'uswds-core/src/styles/mixins/typography/usa-content-styles';
```

Comment:
This stylesheet is essentially [the same as `elements.scss`](https://github.com/department-of-veterans-affairs/component-library/blob/9b902d53335676bd8ed2a96f94b8640d131e3aa6/packages/css-library/src/stylesheets/elements.scss) but I could not move that file because it's being used in the component library and has `rem` values.

This file is the "16px root font" version and `elements.scss` is the "10px root font" version.